packages/remix/src/generators/application/files/common/remix.config.js__tmpl__

```diff
@@ -7,5 +7,5 @@ module.exports = {
   // assetsBuildDirectory: "public/build",
   // serverBuildPath: "build/index.js",
   // publicPath: "/build/",
-  watchPaths: ["<%= offsetFromRoot %>libs"]
-};
\ No newline at end of file
+  watchPaths: () => require("@nx/remix").createWatchPaths(__dirname),
```

Comment:
This require is in the function body so it's not at the top-level of the config. This means it's only needed during development and no for production.